Compiling without corba

toolchain + rtt_ros (2.9) compiled from source **without** CORBA .

It seems that those two lines :
```cmake
find_package(OROCOS-RTT 2.0.0 COMPONENTS rtt-scripting rtt-marshalling rtt-transport-mqueue rtt-transport-corba) # gives OROCOS-RTT_FIND_REQUIRED=
# ... and then 
find_package(catkin REQUIRED COMPONENTS rtt_roscomm kdl_msgs) # gives OROCOS-RTT_FIND_REQUIRED=1, and catches the above rtt-transport-mqueue rtt-transport-corba
```
Results in : 
```
Errors     << rtt_kdl_msgs:cmake /home/hoarau/ws/flr_ws/logs/rtt_kdl_msgs/build.cmake.004.log                                                                               
CMake Error at /home/hoarau/ws/orocos-2.9_ws/install/lib/cmake/orocos-rtt/orocos-rtt-config.cmake:215 (message):
  Could not find the following required OROCOS-RTT plugins:
  rtt-transport-corba.  You may want to append the plugins folder location to
  the RTT_COMPONENT_PATH environment variable.  E.g., if the plugin is
  located at /path/to/plugins/libfoo-plugin.so, add /path/to to
  RTT_COMPONENT_PATH
Call Stack (most recent call first):
  /home/hoarau/ws/rtt_ros-2.9_ws/install/share/rtt_ros/cmake/rtt_ros-extras.cmake:3 (find_package)
  /home/hoarau/ws/rtt_ros-2.9_ws/install/share/rtt_ros/cmake/rtt_ros-extras.cmake:10 (use_orocos)
  /home/hoarau/ws/rtt_ros-2.9_ws/install/share/rtt_ros/cmake/rtt_rosConfig.cmake:197 (include)
  /home/hoarau/ws/rtt_ros-2.9_ws/install/share/rtt_roscomm/cmake/rtt_roscommConfig.cmake:172 (find_package)
  /opt/ros/kinetic/share/catkin/cmake/catkinConfig.cmake:76 (find_package)
  CMakeLists.txt:14 (find_package)
```

I would make use of the `use_orocos()` macro in `rtt_ros` : 
```cmake
cmake_minimum_required(VERSION 2.8.3)
project(rtt_kdl_msgs)

## find catkin and catkin dependencies
find_package(catkin REQUIRED COMPONENTS rtt_ros kdl_msgs)
catkin_destinations()
use_orocos(rtt-scripting)
use_orocos(rtt-marshalling)

## Build options
# mqueue transport
OPTION(ENABLE_MQ "Build posix message queue transport plugin for ${_package}" OFF)
if(ENABLE_MQ)
  use_orocos(rtt-transport-mqueue)
  if (OROCOS-RTT_MQUEUE_FOUND)
    message(STATUS "Building MQueue transport plugin for ROS messages in package ${_package} with ${OROCOS-RTT_MQUEUE_LIBRARIES}")
  else()
    message(WARNING "Disabled built of posix message queue transport plugin for ${_package} because the RTT mqueue plugin could not be found.")
    set(ENABLE_MQ OFF CACHE BOOL "Build posix message queue transport plugin for ${_package} (forced to OFF)" FORCE)
  endif()
endif()

# corba transport
OPTION(ENABLE_CORBA "Build CORBA transport plugin for ${_package}" OFF)
if(ENABLE_CORBA)
  use_orocos(rtt-transport-corba)
  if(OROCOS-RTT_CORBA_FOUND)
    message(STATUS "Building CORBA transport plugin for ROS messages in package ${_package} with ${OROCOS-RTT_CORBA_LIBRARIES}")
  else()
    message(WARNING "Disabled built of CORBA transport plugin for ${_package} because the RTT CORBA plugin could not be found.")
    set(ENABLE_CORBA OFF CACHE BOOL "Build CORBA transport plugin for ${_package} (forced to OFF)" FORCE)
  endif()
endif()
```
And in `package.xml` : 

```xml
<depend>rtt_ros</depend>
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,15 +1,6 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(rtt_kdl_msgs)
 
-## Find Orocos RTT and plugins
-find_package(OROCOS-RTT 2.0.0 COMPONENTS rtt-scripting rtt-marshalling rtt-transport-mqueue rtt-transport-corba)
-if (NOT OROCOS-RTT_FOUND)
-  message(FATAL_ERROR "\n   RTT not found. Is the version correct? Use the CMAKE_PREFIX_PATH cmake or environment variable to point to the installation directory of RTT.")
-else()
-  include(${OROCOS-RTT_USE_FILE_PATH}/UseOROCOS-RTT.cmake)
-  #add_definitions( -DRTT_COMPONENT )
-endif()
-
 ## find catkin and catkin dependencies
 find_package(catkin REQUIRED COMPONENTS rtt_roscomm kdl_msgs)
 catkin_destinations()
@@ -18,8 +9,9 @@ catkin_destinations()
 # mqueue transport
 OPTION(ENABLE_MQ "Build posix message queue transport plugin for ${_package}" OFF)
 if(ENABLE_MQ)
+  use_orocos(rtt-transport-mqueue)
   if (OROCOS-RTT_MQUEUE_FOUND)
-    message(STATUS "Building MQueue transport plugin for ROS messages in package ${_package} with ${OROCOS-RTT_MQUEUE_LIBRARIES}")
+    message(STATUS "Building MQueue transport plugin for ROS messages in package ${PROJECT_NAME} with ${OROCOS-RTT_MQUEUE_LIBRARIES}")
   else()
     message(WARNING "Disabled built of posix message queue transport plugin for ${_package} because the RTT mqueue plugin could not be found.")
     set(ENABLE_MQ OFF CACHE BOOL "Build posix message queue transport plugin for ${_package} (forced to OFF)" FORCE)
@@ -29,8 +21,9 @@ endif()
 # corba transport
 OPTION(ENABLE_CORBA "Build CORBA transport plugin for ${_package}" OFF)
 if(ENABLE_CORBA)
+  use_orocos(rtt-transport-corba)
   if(OROCOS-RTT_CORBA_FOUND)
-    message(STATUS "Building CORBA transport plugin for ROS messages in package ${_package} with ${OROCOS-RTT_CORBA_LIBRARIES}")
+    message(STATUS "Building CORBA transport plugin for ROS messages in package ${PROJECT_NAME} with ${OROCOS-RTT_CORBA_LIBRARIES}")
   else()
     message(WARNING "Disabled built of CORBA transport plugin for ${_package} because the RTT CORBA plugin could not be found.")
     set(ENABLE_CORBA OFF CACHE BOOL "Build CORBA transport plugin for ${_package} (forced to OFF)" FORCE)

